com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -418,7 +418,48 @@ static bool IsMultiPassStereoEnabled(Camera camera)
             return false;
         }
 
-        Comparison<Camera> cameraComparison = (camera1, camera2) => { return (int) camera1.depth - (int) camera2.depth; };
+#if ENABLE_VR && ENABLE_VR_MODULE
+        static List<XR.XRDisplaySubsystem> displaySubsystemList = new List<XR.XRDisplaySubsystem>();
+        static XR.XRDisplaySubsystem GetXRDisplaySubsystem()
+        {
+            XR.XRDisplaySubsystem display = null;
+            SubsystemManager.GetInstances(displaySubsystemList);
+
+            if (displaySubsystemList.Count > 0)
+                display = displaySubsystemList[0];
+
+            return display;
+        }
+
+        // NB: This method is required for a hotfix in Hololens to prevent creating a render texture when using a renderer
+        // with custom render pass.
+        // TODO: Remove this method and usages when we have proper dependency tracking in the pipeline to know
+        // when a render pass requires camera color as input.
+        internal static bool IsRunningHololens(Camera camera)
+        {
+#if PLATFORM_WINRT
+            if (IsStereoEnabled(camera))
+            {
+                var platform = Application.platform;
+                if (platform == RuntimePlatform.WSAPlayerX86 || platform == RuntimePlatform.WSAPlayerARM)
+                {
+                    var displaySubsystem = GetXRDisplaySubsystem();
+                    var subsystemDescriptor = displaySubsystem?.SubsystemDescriptor ?? null;
+                    string id = subsystemDescriptor?.id ?? "";
+
+                    if (id.Contains("Windows Mixed Reality Display"))
+                        return true;
+
+                    if (!XR.WSA.HolographicSettings.IsDisplayOpaque)
```

Comment:
Will this work with OpenXR + HL2? We should discuss this.

Reply:
This "WSA" API won't work for OpenXR, neither will the "string" test above.  However, this [`XRDisplaySubsystem.displayOpaque`](https://docs.unity3d.com/ScriptReference/XR.XRDisplaySubsystem-displayOpaque.html) should work across all XR SDK (both windowsmr, and openxr plugin).

Reply:
Addressed in 
cd10234